### PR TITLE
Added TCS, TSC, TCD, TDC, TXY, TYX in CPU.cpp

### PR DESCRIPTION
**Pull request by arunabh-cloud**

### CODE REVIEW

File: src/core/CPU.cpp [hunk 1]
```diff
@@ -866,22 +866,29 @@ Blaze::Cycles Blaze::CPU::executeTAY() {
 };
 
 Blaze::Cycles Blaze::CPU::executeTCD() {
-	// TODO
+	// TODO TO CHECK --What should be sent to the setZeroNegFlags(--,--) function
+	DR = A;
```
Comment (facekapow) [inline src/core/CPU.cpp:870]:
```suggestion
	DR = A.forceFullLoad();
```

File: src/core/CPU.cpp [hunk 2]
```diff
@@ -866,22 +866,29 @@ Blaze::Cycles Blaze::CPU::executeTAY() {
...
+	setZeroNegFlags(A, true);
 	return 0;
 };
 
 Blaze::Cycles Blaze::CPU::executeTCS() {
-	// TODO
+	// TODO TO CHECK
+	SP = A;
```
Comment (facekapow) [inline src/core/CPU.cpp:877]:
```suggestion
	SP = A.forceFullLoad();
```

File: src/core/CPU.cpp [hunk 3]
```diff
@@ -866,22 +866,29 @@ Blaze::Cycles Blaze::CPU::executeTAY() {
...
 	return 0;
 };
 
 Blaze::Cycles Blaze::CPU::executeTDC() {
-	// TODO
+	// TODO TO CHECK 
+	A = DR;
```
Comment (facekapow) [inline src/core/CPU.cpp:883]:
```suggestion
	A.forceFullStore(DR);
```

File: src/core/CPU.cpp [hunk 4]
```diff
@@ -866,22 +866,29 @@ Blaze::Cycles Blaze::CPU::executeTAY() {
...
+	setZeroNegFlags(A, true);
 	return 0;
 };
 
 Blaze::Cycles Blaze::CPU::executeTSC() {
-	// TODO
+	// TODO TO CHECK 
+	A = SP;
```
Comment (facekapow) [inline src/core/CPU.cpp:890]:
```suggestion
	A.forceFullStore(SP);
```

File: src/core/CPU.cpp [hunk 5]
```diff
@@ -903,7 +910,9 @@ Blaze::Cycles Blaze::CPU::executeTXS() {
 };
 
 Blaze::Cycles Blaze::CPU::executeTXY() {
-	// TODO
+	// TODO TO CHECK 
+	Y = X;
```
Comment (facekapow) [inline src/core/CPU.cpp:914]:
```suggestion
	Y = X.load();
```

File: src/core/CPU.cpp [hunk 6]
```diff
@@ -914,7 +923,9 @@ Blaze::Cycles Blaze::CPU::executeTYA() {
 };
 
 Blaze::Cycles Blaze::CPU::executeTYX() {
-	// TODO
+	// TODO TO CHECK 
+	X = Y;
```
Comment (facekapow) [inline src/core/CPU.cpp:927]:
```suggestion
	X = Y.load();
```

**Review by facekapow — CHANGES_REQUESTED** [review]
Looks good, just needs to be updated for my changes on the branch.

**Comment by arunabh-cloud** [comment]
Done.

File: src/core/CPU.cpp [hunk 7]
```diff
@@ -862,22 +862,26 @@ Blaze::Cycles Blaze::CPU::executeTAY() {
...
-	// TODO
+	// TODO TO CHECK --What should be sent to the setZeroNegFlags(-) function
+	DR = A.forceFullLoad();
+	setZeroNegFlags(A);
 	return 0;
 };
 
 Blaze::Cycles Blaze::CPU::executeTCS() {
-	// TODO
+	SP = A.forceFullLoad();
```
Comment (facekapow) [inline src/core/CPU.cpp:872]:
```suggestion
	SP = A.forceLoadFull();
```

File: src/core/CPU.cpp [hunk 8]
```diff
@@ -862,22 +862,26 @@ Blaze::Cycles Blaze::CPU::executeTAY() {
 };
 
 Blaze::Cycles Blaze::CPU::executeTCD() {
-	// TODO
+	// TODO TO CHECK --What should be sent to the setZeroNegFlags(-) function
+	DR = A.forceFullLoad();
```
Comment (facekapow) [inline src/core/CPU.cpp:866]:
```suggestion
	DR = A.forceLoadFull();
```

File: src/core/CPU.cpp [hunk 9]
```diff
@@ -862,22 +862,26 @@ Blaze::Cycles Blaze::CPU::executeTAY() {
...
 
 Blaze::Cycles Blaze::CPU::executeTCS() {
-	// TODO
+	SP = A.forceFullLoad();
 	return 0;
 };
 
 Blaze::Cycles Blaze::CPU::executeTDC() {
-	// TODO
+	A.forceFullStore(DR);
```
Comment (facekapow) [inline src/core/CPU.cpp:877]:
```suggestion
	A.forceStoreFull(DR);
```

File: src/core/CPU.cpp [hunk 10]
```diff
@@ -862,22 +862,26 @@ Blaze::Cycles Blaze::CPU::executeTAY() {
...
+	setZeroNegFlags(A);
 	return 0;
 };
 
 Blaze::Cycles Blaze::CPU::executeTSC() {
-	// TODO
+	A.forceFullStore(SP);
```
Comment (facekapow) [inline src/core/CPU.cpp:883]:
```suggestion
	A.forceStoreFull(SP);
```

**Review by facekapow — CHANGES_REQUESTED** [review]
Sorry, my bad; I got the method name wrong. 😓

**Comment by arunabh-cloud** [comment]
It happens to me all the time! :)

**Pull request merged**